client/app/services/intercom.js

```diff
@@ -23,7 +29,16 @@ export default Ember.Service.extend({
     /* eslint-enable camelcase */
   },
 
+  trackEvent(eventName) {
+    let eventAttribute = Ember.String.camelize(eventName);
+    if (!this.get(eventAttribute))  {
```

Comment:
Sure. It just seems like "we want to track this once per session" is a detail that happens to be true for the events you're implementing here, but might not be true for future ones. I guess we could handle that conflict if and when it occurs (maybe a distinction between `trackEvent` and `trackEventOnce` or something).

Regarding the approach, what do you think about putting an explicit hash property on the service, e.g. `trackedEvents`, that stores which events have already been tracked? That might better reveal the intent of this code.